css/properties/display.json

```diff
@@ -119,30 +121,34 @@
                   "version_added": "8"
                 },
                 "opera": {
-                  "version_added": true,
-                  "version_removed": "19"
+                  "version_added": "7",
+                  "version_removed": "15"
```

Comment:
I believe that "19" **is** the correct value here based upon the Chrome version (32 - 13 = 19).  Perhaps we should put 19 back?

Reply:
Oh good catch! Fixing now!